### ISSUE TITLE
Discover how many volunteers have accessibility issues

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1346]
to do once we have finished accessibility improvements

**Comment by AlOneill** [comment]
If this is to collect baseline data, so that we can assess whether or not we are attracting more volunteers in the future who would have difficulty accessing a site that did not address the fundamental accessibility issues, then all well and good.

Bear in mind that any markup issues *specific* to the Members' pages (and there are currently quite a few of these) will not be covered by the initial improvements identified by the Shaw Trust.

**Comment by PatReynolds** [comment]
This was a story requested by (I think) EricB.  You are correct, it does need to have a purpose attached to it. @edickens can you comment?

**Comment by edickens** [comment]
Ever since we started with the new FR the transcribers and coordinators have asked for less "air" and better contrast, especially on the listings.  So, if we do not need the wide spacing for those who have logged in, then could we have a different style for these screen?  We can still have the facilities for those who cannot use a mouse.  I really don't think that someone with severe visual problems could transcribe.  But perhaps they can?  We don't know whether we are doing a lot of work, which will never be used, to the detriment of those who would like less scrolling and more on the screen at one time.
We might also like to know, from those who have logged in, if they do use their phones   I know some use tablets.  EricD

**Comment by Sherlock21** [comment]
Indeed this question was raised by me. it arose from the proposal to change colours throughout the FR application

The reason is quite obvious to me - in a world where we are so short of effort, and such a high demand for various activities to be addressed, its clear that we ( collectively) should be spending our time and money on the things which have the greatest benefit to most users.
Its no good spending a lot of effort on an issue which benefits just one or two researchers, but upsets a couple of dozen key users like CC's or DM's, is it? When that same effort could be spent of tasks which benefit all CC's for example, since without those Volunteers, there will be no future for FR.

**Comment by Sherlock21** [comment]
I support Eric D's comment too.
The issue is the same overall problem as I refer to in my comment above.: who and what role do these 'requesters of change' play in FR?

If just researchers then their wish should be well below that of the volunteers who actually do all  the work to keep FR full of data - i.e. CC's and Managers - and Transcribers if they upload and manage their now batches online.
As one of those  active Volunteers who do all these 3 rolls, I agree with Eric D: you need a computer and mouse for best and most accurate results - as its just too hard with scrolling on a smart phone or tablet - and I have tried both.

**Comment by edickens** [comment]
May I add, that when we get to the stage where researchers can register and log in so that they can get access to more detail and leave requests to be informed of changes, then their Actions will need to conform to the standards for visually impaired.  But it will be for just a few Actions and they will most likely be new ones.  EricD

**Comment by PatReynolds** [comment]
It is already our policy NOT to implement modifications that would permit the use of transcription software and transcription-related website areas which would not benefit any (potential) volunteers because of the sight needed to transcribe.

The changes we would be making to bring the researcher interfaces up to AA standard will benefit many people. I'm finding it difficult to find an source that confirms how many people use voice commands because of disability in general or dexterity specifically - but it appears that 2,856,900 people in the UK have dexterity problems - 4.35% (probably around 15%+ in the older age groups who are most represented among both users and volunteers). 

Requesters of change: anyone can request change.  We sort these requests every fortnight in a pre-scrum meeting. I represent the Trustees, Rich and Vino the developers, and Steve and Eric D the project management team.

**Comment by Captainkirkdawson** [comment]
There are no development activities associated with this. It is a management survey
